src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -221,6 +221,7 @@ private static void RegisterCommonOptions(Command command)
             command.AddOption(new Option<IReadOnlyCollection<string>>(new[] { "--extension" }, "Specifies a .NET Upgrade Assistant extension package to include. This could be an ExtensionManifest.json file, a directory containing an ExtensionManifest.json file, or a zip archive containing an extension. This option can be specified multiple times."));
             command.AddOption(new Option<IReadOnlyCollection<string>>(new[] { "--entry-point", "-e" }, "Provides the entry-point project to start the upgrade process. This may include globbing patterns such as '*' for match."));
             command.AddOption(new Option<UpgradeTarget>(new[] { "--target-tfm-support" }, "Select if you would like the Long Term Support (LTS), Current, or Preview TFM. See https://dotnet.microsoft.com/platform/support/policy/dotnet-core for details for what these mean."));
+            command.AddOption(new Option<bool>(new[] { "--readiness-acknowledgement", "-ra" }, "Acknowledges that project(s) contain technologies that are not supported."));
```

Comment:
Why is this an option for the end user? I wouldn't know what/when to supply this as a user

Reply:
renamed to `--ignore-unsupported-features`